packages/eslint-plugin-office-addins/src/utils/load.ts

```diff
@@ -41,28 +41,46 @@ function composeObjectExpressionPropertyIntoString(
   return composedProperty;
 }
 
+function parseLoadStringArgument(argument: string): string[] {
+  let properties: string[] = [];
+  argument
+    .replace(/\s/g, "")
+    .split(",")
+    .forEach((property: string) => {
+      properties.push(property);
+    });
+
+  return properties;
+}
+
 export function parseLoadArguments(node: TSESTree.MemberExpression): string[] {
   node = findTopLevelExpression(node);
 
   if (
     isLoadFunction(node) &&
     node.parent?.type === TSESTree.AST_NODE_TYPES.CallExpression
   ) {
+    const argument = node.parent.arguments[0];
+    if (!argument) return [];
+
     let properties: string[] = [];
-    node.parent.arguments.forEach(
-      (argument: TSESTree.CallExpressionArgument) => {
-        if (argument.type === TSESTree.AST_NODE_TYPES.Literal) {
-          (argument.value as string)
-            .replace(/\s/g, "")
-            .split(",")
-            .forEach((property: string) => {
-              properties.push(property);
-            });
-        } else if (argument.type === TSESTree.AST_NODE_TYPES.ObjectExpression) {
-          properties.push(composeObjectExpressionPropertyIntoString(argument));
+    if (argument.type === AST_NODE_TYPES.ArrayExpression) {
+      argument.elements.forEach((args) => {
+        if (args.type === TSESTree.AST_NODE_TYPES.Literal) {
+          properties = properties.concat(
+            parseLoadStringArgument(args.value as string)
+          );
         }
+      });
+    } else {
+      if (argument.type === TSESTree.AST_NODE_TYPES.Literal) {
+        properties = properties.concat(
+          parseLoadStringArgument(argument.value as string)
+        );
```

Comment:
This is handling the same thing as an element in an array . . . should share logic if you can.

Reply:
This is just one line of code, it is just that the linter makes it 3 lines.
I don't quite think we need to share the logic, it just appears complex because the commands themselves are lengthy.

Reply:
I was referring to the whole if statement not just the method call.  

I was thinking that each element in the array has the same possibilities as an individual non-array load argument and therefore you could process an element the same as a non-array load argument.  But if that isn't the case (see the comment about the object as an array element) then it does become less compelling

Note that the one method is only because you moved that chunk of code to a method so you could use it in both locations.  It if made sense to move the if statements there too if would help to reduce the nesting of if, else if, and forEach code that gets a little tricky to follow.  But that depends on the other comment about Objects being valid in the array.

Reply:
Objects are not valid, so I guess we don't have much to change about this.